### ISSUE TITLE
Compiling a list of words that have entries in other major dictionaries but not in jmdict

### ISSUE THREAD

**Issue by Marcusjmdict** [JMdictProject/JMdictIssues#52]
Looking at the transitivity lists Jim were able to compile from a Meikyo EB file, I was wondering if it might be possible to compile a list of all words in various other dictionaries' Epwing files (etc.) that don't yet have entries in JMdict? If it were possible to sort these "missing entries" after the number of dictionaries they appear in, such a list could become a very useful tool for expanding JMdict.

**Comment by robinjmdict** [comment]
I don't think this would be helpful unless we ordered them by frequency (using the n-gram data).

Daijrin/Daijisen/Kojien have 250-300k entries. I'd estimate that at least half of them aren't in JMdict. The vast majority will be archaic and/or obscure.

Meikyo and Shinmeikai (which omit most archaic words) have 70-80k entries, but again, most of the missing ones will be very obscure.

At this point, I don't think the kokugos are the best place to go looking for more terms.

**Comment by Marcusjmdict** [comment]
Right, it depends on how much work it would be, if it's something that would take a great deal of time to set up it might not be worth it. It really is a big shame there's no easy way to filter out the tens of thousands of archaic words in the major kokugos. But, if we included GG5, Jitsuyo and Wadoku (and Eijiro, Chiezo, Nipponica and Britannica etc. if these files are available to us) and maybe weighted daijs, daijr, koj and nikk as somewhere between 0.25 and 0.5 each (as opposed to 1 for each appearance of a word in any other reference), I think the resulting list could be interesting and useful, even without the ngrams data.

**Comment by JMdictProject** [comment]
I've done a bit in this area in the past and have some tools and code I can recycle. Also I am able to run bulk n-gram extractions without having to go through the WWW interface.

I'm basically out of time to look at it this year, but I should be able to do something in February. I'd like to be able to build lists of non-JMdict common surface forms which are in the major references. A good goal would be to have them in a WWW framework where it would be easy to initiate entry creation. I should also be able to refresh it regularly so the the list gets culled as entries get added.

Anyway, I'll level the issue open for now.

**Comment by Marcusjmdict** [comment]
One thing that just struck me be helpful to identify modern words in Daijisen and Daijirin is to check for words that are marked as スル, I don't think they use them for archaic words (at least not to a large extent)
